centos_ci.sh

```diff
@@ -180,24 +181,18 @@ function artifacts_upload_on_pr_and_master_trigger() {
   set +x
   # For PR build, GIT_BRANCH is set to branch name other than origin/master
   if [[ "$GIT_BRANCH" = "origin/master" ]]; then
-    install_docs_prerequisite_packages;
-    make gen_adoc_tar IMAGE_UID=$(id -u)
-    build_openshift_origin_docs $(pwd)/docs/build/minishift-adoc.tar;
-
-    mkdir -p minishift/master/$BUILD_NUMBER/
+    mkdir -p minishift/master/$BUILD_NUMBER/ minishift/master/$LATEST
     cp -r out/*-amd64 minishift/master/$BUILD_NUMBER/
-    # Copy the openshift-docs
-    cp -r openshift-docs/_preview/openshift-origin minishift/master/$BUILD_NUMBER/openshift-docs
+    cp -r out/*-amd64 minishift/master/$LATEST/
```

Comment:
This is not symlink, this is more like you are copy bits over to latest folder.

Reply:
This means it is harder to track the origin of the 'latest'...

Reply:
@gbraad , why you want to track origin of the latest? on every PR merge, this means the artifacts of latest merged PR.

@praveenkumar , I need to copy since I found during implementatio of [doc tar upload](https://github.com/minishift/minishift/blob/master/centos_ci.sh#L205-L212) that symlink is not working efficiently with rsync. We rsync content from build machine to artifact server.

Reply:
If you want someone to use the `latest`, how do we really know what it pointed to at that moment? You could at that moment only know from running the executable...

I start to doubt if `latest` makes sense in the context of PR.